front/src/modules/apollo/optimistic-effect/states/optimisticEffectState.ts

```diff
@@ -0,0 +1,10 @@
+import { atom } from 'recoil';
+
+import { OptimisticEffect } from '../types/OptimisticEffect';
+
+export const optimisticEffectState = atom<
+  Record<string, OptimisticEffect<unknown, unknown>>
+>({
+  key: 'optimisticEffectState',
+  default: {},
```

Comment:
@charlesBochet Don't we need to keep optimistic effect state in local storage, as Apollo cache is stored inside it ?

Reply:
I've found no official way to access the list of queries through apollo API, so I prefer to maintain my own state, rather that to take a dependency on something that won't be stable